### PR TITLE
lkl: Adding comment to lkl_dev_net_ops

### PR DESCRIPTION
**Pull request by liuyuan10**
Adding comments to make it clear what is expected to the interfaces of
lkl_dev_net_ops.

Signed-off-by: Yuan Liu liuyuan@google.com

---

This change is [Reviewable](https://reviewable.io/reviews/lkl/linux/141)

### CODE REVIEW

**Comment by pscollins** [comment]
There are spaces where there should be tab characters in a few places, take a look. Good idea to add this.

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[tools/lkl/include/lkl_host.h, line 58 \[r1\]](https://reviewable.io:443/reviews/lkl/linux/141#-KE8G2QCg_zO_zIQ68at:-KE8G2QD3OAtZx7vWM4v:194332037) ([raw file](https://github.com/lkl/linux/blob/8dac72e298140f0847d96aec4942d3fdeeb4826e/tools/lkl/include/lkl_host.h#L58)):*
I think you want to say something like "the data buffer can only hold 0 or 1 complete packets" since it can be the case that len == 0

---

*[tools/lkl/include/lkl_host.h, line 98 \[r1\]](https://reviewable.io:443/reviews/lkl/linux/141#-KE8H2SbxsN4kD94ZnEQ:-KE8H2SbxsN4kD94ZnER:883674733) ([raw file](https://github.com/lkl/linux/blob/8dac72e298140f0847d96aec4942d3fdeeb4826e/tools/lkl/include/lkl_host.h#L98)):*
You may want to add that it must be safe to call free_mem after this returns success

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/lkl/linux/141#-:-KE8GTit05fUwcW_V7HK:456933689)*

**Comment by liuyuan10** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[tools/lkl/include/lkl_host.h, line 58 \[r1\]](https://reviewable.io:443/reviews/lkl/linux/141#-KE8G2QCg_zO_zIQ68at:-KE8MdqVlfFTMoeOgbkY:-499146009) ([raw file](https://github.com/lkl/linux/blob/8dac72e298140f0847d96aec4942d3fdeeb4826e/tools/lkl/include/lkl_host.h#L58)):*
Done.

---

*[tools/lkl/include/lkl_host.h, line 98 \[r1\]](https://reviewable.io:443/reviews/lkl/linux/141#-KE8H2SbxsN4kD94ZnEQ:-KE8N8cKmuIaRuYQcEwu:-499146009) ([raw file](https://github.com/lkl/linux/blob/8dac72e298140f0847d96aec4942d3fdeeb4826e/tools/lkl/include/lkl_host.h#L98)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/lkl/linux/141)*

**Comment by pscollins** [comment]
LGTM

File: tools/lkl/include/lkl_host.h [hunk 1]
```diff
@@ -53,14 +53,59 @@ struct lkl_netdev {
...
 struct lkl_dev_net_ops {
+	/* Writes a L2 packet into the net device.
+	 *
+	 * The data buffer can only hold 0 or 1 complete packets.
+	 *
+	 * @nd - pointer to the network device
+	 * @data - pointer to the buffer
+	 * @len - size of the buffer in bytes
+	 * @returns 0 for success and -1 for failure.
+	*/
```
Comment (stfairy) [inline tools/lkl/include/lkl_host.h]: need one more space before *

Comment (liuyuan10) [inline tools/lkl/include/lkl_host.h]: DONE

**Pull request merged**